internal/lexer/gen/golang/lexer.go

```diff
@@ -180,39 +180,39 @@ func (this *Lexer) Scan() (tok *token.Token) {
 
 			switch rune1 {
 			case '\n':
-				this.line++
-				this.column = 1
+				l.line++
+				l.column = 1
 			case '\r':
-				this.column = 1
+				l.column = 1
 			case '\t':
-				this.column += 4
+				l.column += 4
 			default:
-				this.column++
+				l.column++
 			}
 
 			switch {
 			case ActTab[state].Accept != -1:
 				tok.Type = ActTab[state].Accept
 				// fmt.Printf("\t Accept(%s), %s(%d)\n", string(act), token.TokMap.Id(tok), tok)
```

Comment:
If I remember correctly these lines have been broken since before the line col fix.